src/ui/pages/GenericInfoPage.tsx

```diff
@@ -34,6 +34,17 @@ export const GenericInfoPage: FunctionComponent<Props> = ({
   flex = true,
   buttons,
 }) => {
+  const timeRef = useRef(0)
+  useEffect(() => {
+    timeRef.current = setTimeout(() => {
+      StatusBar.setBarStyle('light-content', true)
+    })
+    return () => {
+      clearTimeout(timeRef.current)
+      StatusBar.setBarStyle('dark-content', true)
+    }
+  }, [])
```

Comment:
on le fait à 2 endroits, on pourrait en faire un hook

t'en penses quoi ?

Reply:
un hook avec un argument `noBackground` (qui serait optionel avec une valeur par défaut)

Reply:
Don’t rush to add Effects to your components. Keep in mind that Effects are typically used to “step out” of your React code and synchronize with some external system. This includes browser APIs, third-party widgets, network, and so on. If your Effect only adjusts some state based on other state, [you might not need an Effect.](https://beta.reactjs.org/learn/you-might-not-need-an-effect)

Sans remettre en question le code ci-dessus je me demande si il n'est pas un bon cas pour "ne pas avoir d'effet" ?